test/lightning/impact_tracking/worker_test.exs

```diff
@@ -1,12 +1,15 @@
 defmodule Lightning.ImpactTracking.WorkerTest do
-  use ExUnit.Case, async: false
+  use Lightning.DataCase, async: false
 
-  alias Lightning.ImpactTracking.{Client, Worker}
+  alias Lightning.ImpactTracking.{Client, Configuration, Report, Worker}
 
   import Mock
   import Lightning.ApplicationHelpers, only: [put_temporary_env: 3]
 
   @host "https://foo.bar"
+  @metrics %{}
+  @ok_response {:ok, @metrics}
+  @error_response {:error, @metrics}
```

Comment:
this is a nit pick and probably you have used it in multiple places before. Now that these are used only on the mocking template functions, would just set the return there to be `{:ok, @metrics}` and `{:error, @metrics}`